### ISSUE TITLE
[PYTHON] Code not valid for Python 2.6

### ISSUE THREAD

**Issue by hmguardado** [swagger-api/swagger-codegen#4084]
##### Description

When I generate the code for the client from the server description, there are some functions that not exist in Python 2.6 such us:

```
        # query parameters
        if query_params:
            query_params = self.sanitize_for_serialization(query_params)
            query_params = {k: self.to_path_value(v)
                            for k, v in iteritems(query_params)}
```

Is there any valid way to auto-generate code for Python 2.6 with your library?

Thanks in advance,

Héctor Martínez 
##### Swagger-codegen version

2.2.1
##### Swagger declaration file content or url

Sorry, I cannot post it.
##### Command line used for generation

java -jar swagger-codegen-cli-2.2.1.jar generate -i "C:\Swagger\docs\my_swagger.json" -l python -o C:\Swagger\my_swagger_client
##### Steps to reproduce

When I try to test the api with some test functions that are also autegerated. Test files in python executed with C:\Python26\python.exe

**Comment by wing328** [comment]
```
        # query parameters
        if query_params:
            query_params = self.sanitize_for_serialization(query_params)
            query_params = {k: self.to_path_value(v)
                            for k, v in iteritems(query_params)}
```

Thanks for reporting the issue. How could the above be revised to support Python 2.6?

cc @scottrw93

**Comment by pryankster** [comment]
```python

if query_params:
    query_params = self.sanitize_for_serialization(query_params)
    query_params = dict( [ (kv[0],self.to_path_value(kv[1])) for kv in query_params.iteritems() ] )
```
Should work.  Instead of dictionary comprehension, which is 2.7+, it uses a dictionary constructor taking a list of tuples.

--pryankster

**Comment by wing328** [comment]
@pryankster we've released 2.2.2: https://github.com/swagger-api/swagger-codegen/releases/tag/v2.2.2

Please give it a try and let us know if you still experience issues using Python 2.6
